### PR TITLE
use HBHEPhase1Reconstructor for NZS

### PR DESCRIPTION
**Pull request by kpedro88**
Followup to #17688, propagating new reconstructor to the HCAL NZS sequence.

attn: @igv4321, @abdoulline, @bsunanda

### CODE REVIEW

**Comment by kpedro88** [comment]
@cmsbuild please test

**Comment by perrotta** [comment]
+1
Tested with wf 136.731 with the replacement RECO to RECO:reconstruction_HcalNZS
No difference with baseline, as expected since hcalLocalRecoNZS uses only HCAL method 0, and with the the new reconstructor "there are essentially no differences between Run 2 and Phase 1 rechit energies reconstructed by Method 0" (https://github.com/cms-sw/cmssw/pull/17688#issue-211376346)

**Comment by davidlange6** [comment]
+1

File: RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py [hunk 1]
```diff
@@ -1,40 +1,38 @@
 import FWCore.ParameterSet.Config as cms
 
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
-hbherecoMB = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
+import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
+hbherecoMB = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
+    dropZSmarkedPassed = cms.bool(False),
```
Comment (slava77) [inline RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py]: was .clone not enough , else how to explain this problem 
https://hypernews.cern.ch/HyperNews/CMS/get/recoDevelopment/1522.html

Comment (kpedro88) [inline RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py]: @slava77 I'm looking into it.

**Pull request merged**